tests/e2e/tests/boosts.spec.ts

```diff
@@ -143,4 +144,64 @@ describe('Boosts', () => {
         const children = await a.invoke.getBoostChildren(parentBoostUri);
         expect(children.records).toHaveLength(0);
     });
+
+    test('Users can send a boost via signing authority using the HTTP route', async () => {
+        const learnCard = await getLearnCard(crypto.randomBytes(32).toString('hex'));
+        await learnCard.invoke.createServiceProfile({
+            profileId: 'rando',
+            displayName: 'Random User',
+            bio: '',
+            shortBio: '',
+        });
+        // Create and register a signing authority for user A
+        const sa = await learnCard.invoke.createSigningAuthority('test-sa');
+        expect(sa).toBeDefined();
+        await learnCard.invoke.registerSigningAuthority(sa.endpoint, sa.name, sa.did);
+        const saResult = await learnCard.invoke.getRegisteredSigningAuthority(sa.endpoint, sa.name);
+        expect(saResult).toBeDefined();
+        const signingAuthority = {
+            endpoint: saResult.signingAuthority.endpoint,
+            name: saResult.relationship.name,
+        };
+
+        // Create a boost as user A
+        const boostUri = await learnCard.invoke.createBoost(testUnsignedBoost);
+        expect(boostUri).toBeDefined();
+
+        const grantId = await learnCard.invoke.addAuthGrant({
+            status: 'active',
+            id: 'test',
+            name: 'test',
+            challenge: 'auth-grant:test-challenge',
+            createdAt: new Date().toISOString(),
+            scope: '*:*',
```

Comment:
```suggestion
            scope: 'boosts:write',
```

Not necessary, but if we want to test the auth scope, this makes it tighter?